updates to demo use of meanParentAge

silastittes opened the pull request:
Edits to use the new `meanParentAge` property introduced in SLiM 4.0.1, with minor edits in body of text to explain the update.

silastittes committed:
updates to demo use of meanParentAge

petrelharp commented:
This is great, Silas. I was hoping you might stick in the comparison of site-diversity to branch-length-diversity here as well somehow, with some explanation? (before the 'recapitation' bit)  That would make things more concrete?

petrelharp commented:
(Also see the errors in building the docs.)

silastittes commented:
DOH! I think I just forgot the `git add` the new .trees file.  
But thanks! And I agree adding the branch length comparisons makes sense, will make those updates soon!

silastittes committed:
update cache to trigger slim update

silastittes committed:
dealing with merge conflicts

silastittes commented:
I updated the cache value and added some site-diversity to branch-length-diversity details. Hopefully looks good and  build docs works! <fingers crossed>

petrelharp commented:
Weird I have to keep clicking "approve" on your checks - I will look into this later.

petrelharp committed:
a bit more explanation

petrelharp commented:
Great! I put in a bit more explanation (e.g., something about *why* that check works), and removed the need to re-simulate without mutations - what do you think?

silastittes commented:
Awesome! Looks great and helped to clarify several ideas I was unsure about.
